test/images/Makefile

```diff
@@ -12,19 +12,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-.PHONY: all image push clean
+REGISTRY ?= gcr.io/k8s-e2e-images
```

Comment:
per some of the discussion around the ci images repo, I finally settled on `gcr.io/kubernetes-e2e-test-images`, so we should update to that.

I agree that our version numbering is pretty inconsistent. (the addons are just as bad.) I guess we can start over, for consistency in the `kubernetes-e2e-test-images` registry, but I don't have a particularly strong opinion on which style to use (e.g. `v1, v2, v3, ...` or `0.1, 0.2, 0.3, ...` or `1.0, 1.1, 1.2, ...` or `v1.0.0` or `$(git rev-parse HEAD)` or `blue, red, green, ...` or ...).
Semantic versioning probably means less for e2e test images, though.